lib/tasks/scripts/split_bathinda_and_mansa.rb

```diff
@@ -0,0 +1,34 @@
+module SplitBathindaAndMansa
+  def self.call
+    ActiveRecord::Base.transaction do
+      bathinda_and_mansa = FacilityGroup.find_by!(name: "Bathinda and Mansa")
+
+      # Create these facility groups in IHCI before running this rake task
```

Comment:
Consider `These facility groups need to be created`. Otherwise it's kinda misleading, as if the code below does this.